frontend/src/components/GeyserStakeView.tsx

```diff
@@ -60,22 +44,23 @@ export const GeyserStakeView: React.FC<Props> = () => {
   }
 
   return (
-    <div className="p-4 flex flex-col">
-      {parsedAmount.isZero() ? (
-        <>
-          <div>Wallet balance: {formatDisplayAmount(walletAmount)}</div>
-          <div>Current stake: {formatDisplayAmount(userStake)}</div>
-        </>
-      ) : (
-        <>
-          <div>New wallet balance: {formatDisplayAmount(walletAmount.sub(parsedAmount))}</div>
-          <div>New stake: {formatDisplayAmount(userStake.add(parsedAmount))}</div>
-        </>
-      )}
+    <GeyserStakeViewContainer>
+      <WalletBalance
+        parsedAmount={parsedAmount}
+        walletAmount={walletAmount}
+        decimals={stakingTokenDecimals}
+        symbol={stakingTokenSymbol}
+      />
```

Comment:
I think we will need to also change this to show wallet balance when staking and staked balance when unstaking right? We can fix it later, but maybe we can put a TODO somewhere